c_src/TLH_absTab.c

```diff
@@ -0,0 +1,225 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include <glib.h>
+#include "absTab.h"
+#include "message.h"
+
+struct AbsTab {
```

Comment:
I feel like this struct name doesn't properly reflect what this data structure does. At a conceptual level, what this component tries to do maintain a set of pending notifications so I think the interface could be reworked a little.

There's a couple of approaches that I can think of:
- Treat it like a queue and name it `event_queue` and use verbs like `push`/`pop`. This would be fairly intuitive but would hide the fact that we drop duplicate events.
- Treat it like a set and name it `notification_set` or `event_set` and use verbs like `put`/`next`. It's a bit trickier to come up with the right verbs for the functions on this one but it might be overall better.